sdk/keyvault/azure-keyvault-certificates/HISTORY.md

```diff
@@ -1,11 +1,11 @@
 # Release History
 
-# 4.0.0b7 Unreleased
+## 4.0.0b7 (Unreleased)
 - Challenge authentication policy preserves request options
 ([#8999](https://github.com/Azure/azure-sdk-for-python/pull/8999))
 
 
-### 4.0.0b6
+## 4.0.0b6
```

Comment:
Please add the date here, you can find them in PyPI:
https://pypi.org/project/azure-keyvault-certificates/#history